docs/types.rst

```diff
@@ -389,6 +389,14 @@ There are two ways to access a function in the current contract: Either directly
 by its name, ``f``, or using ``this.f``. The former will result in an internal
 function, the latter in an external function.
 
+A function type A is implicitly convertible to a function type B if and only if
```

Comment:
While not necessary, I think consistently marking `A` and `B` with a different font (italic or code highlight aka backticks) could help readability.